packages/sign_in_with_apple/lib/src/sign_in_with_apple.dart

```diff
@@ -203,18 +199,14 @@ class SignInWithApple {
       queryParameters: <String, String>{
         'client_id': webAuthenticationOptions.clientId,
         'redirect_uri': webAuthenticationOptions.redirectUri.toString(),
-        'scope': scopes
-            .map((scope) {
-              switch (scope) {
-                case AppleIDAuthorizationScopes.email:
-                  return 'email';
-                case AppleIDAuthorizationScopes.fullName:
-                  return 'name';
-              }
-              return null;
-            })
-            .where((scope) => scope != null)
-            .join(' '),
+        'scope': scopes.map((scope) {
+          switch (scope) {
+            case AppleIDAuthorizationScopes.email:
+              return 'email';
+            case AppleIDAuthorizationScopes.fullName:
```

Comment:
Oh does this mean switch cases are exhaustive when used in combination with enums? 🤯